packages/blocks/github-pr-overview/src/app.tsx

```diff
@@ -1,241 +1,296 @@
 import * as React from "react";
 
-import { BlockComponent } from "blockprotocol/react";
-import CircularProgress from "@mui/material/CircularProgress";
+import { BlockComponent, useGraphBlockService } from "@blockprotocol/graph";
+import { Box, CssBaseline, Theme, ThemeProvider } from "@mui/material";
 import { theme } from "@hashintel/hash-design-system";
-import { CssBaseline, ThemeProvider } from "@mui/material";
 import {
-  GithubIssueEvent,
-  GithubPullRequest,
   PullRequestIdentifier,
-  GithubReview,
-  isDefined,
-  GITHUB_ENTITY_TYPES,
-  getGithubEntityTypes,
+  BlockState,
+  LocalState,
+  Actions,
 } from "./types";
 import { GithubPrOverview } from "./overview";
-import {
-  collectPrEventsAndSetState,
-  collectPrsAndSetState,
-  collectReviewsAndSetState,
-  getPrs,
-} from "./entity-aggregations";
-import { PullRequestSelector } from "./pull-request-selector";
 
-export enum BlockState {
-  Loading,
-  Error,
-  Selector,
-  Overview,
-}
+import { getPRDetails, getEntityTypeIdsAndPRs } from "./entity-aggregations";
+import { PullRequestSelector } from "./pull-request-selector";
+import { InfoUI } from "./info-ui";
 
 type BlockEntityProperties = {
   selectedPullRequest?: PullRequestIdentifier;
 };
 
+const customTheme: Theme = {
+  ...theme,
```

Comment:
This shows how we can customise the theme exported from our design system package